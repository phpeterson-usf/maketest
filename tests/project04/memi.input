-h $(repo_dir)/memi.hex $(TESTS_DIR)/memi.s
