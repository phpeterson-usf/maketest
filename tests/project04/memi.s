main:
    ldr r2, [r3]
    str r2, [sp, #4]
    bx lr
