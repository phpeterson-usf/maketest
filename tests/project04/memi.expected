v2.0 raw
e5932000
e58d2004
e12fff1e
